app.js

```diff
@@ -319,7 +319,7 @@ const loadTcpServer = net.createServer(function (socket) {
 
   if (STATE === 'up' && Settings.internal.load_balancer_agent.report_load) {
     let availableWorkingCpus
-    const currentLoad = os.loadavg()[0]
+    const currentLoad = os.loadavg()[1]
```

Comment:
I suspect that this change will cause even more clsi nodes to get overloaded.

New projects should not get assigned to a VM already running 16+ compiles.

Reply:
that's fair, lets change 1 thing at a time.